contribs/discrete_mode_choice/src/main/java/org/matsim/contribs/discrete_mode_choice/model/utilities/MaximumSelector.java

```diff
@@ -33,7 +37,7 @@ public Optional<UtilityCandidate> select(Random random) {
 
 	public static class Factory implements UtilitySelectorFactory {
 		@Override
-		public UtilitySelector createUtilitySelector() {
+		public UtilitySelector createUtilitySelector(Person person, List<DiscreteModeChoiceTrip> tourTrips) {
 			return new MaximumSelector();
```

Comment:
Why are the `person` and `tourTrips` not passed to the `MaximumSelector`? That one should also record utilities if desired. The same for all the other possible selectors: i.e.,`RandomSelector`